### ISSUE TITLE
icon image for both ios (not transparent) and android (transparent)

### ISSUE THREAD

**Issue by kctang** [lpender/meteor-assets#6]
Having some issues using a single source icon that works smoothly for both iOS and Android.
1. To ensure that app icons in Android shows up nicely as notification icons, its [background should be transparent](http://stackoverflow.com/questions/28387602/notification-bar-icon-turns-white-in-android-5-lollipop).
2. For iOS, transparent icon background is not recommended to avoid [black background](http://stackoverflow.com/questions/22858501/ios-app-icon-with-transparent-background-showing-black-background-on-device).

Right now, it seems we can only have one or the other...and need to use separate icons based on platform. Any solution or thoughts on this?

**Comment by lpender** [comment]
Thanks for pointing this out.

For the time being it seems like you'd have to hack around it.

Run it once with a non-transparent background. 

Copy the the resources directory. 

Then run it with a transparent background. 

Copy the files in the resources directory (inside `splashes` and `icons`) that start with `android` the corresponding target directory in your app.

**Comment by lpender** [comment]
@kctang care to have a look at this pull request? https://github.com/lpender/meteor-assets/pull/7

Happy to merge it in but a second set of eyes would be helpful.

**Comment by kctang** [comment]
@lpender Sure. I just tried out icons generated by #7 on my app - works! 

Other comments/observations that should not prevent #7 from being merged.:
- Would be good if the bundled `icon-android.png` has transparent background so that when used as notification icon, it does not looks like a "white square".
- The bundled `splash-android.png` is too "wide" for Android portrait layout (on my One Plus One).
- For my use case, I did not use separate splash images for Android & iOS. Is it really needed?

Thanks for this convenient tool.
